charts/flagsmith/templates/NOTES.txt

```diff
@@ -24,6 +24,33 @@ for information about how to gain web access to the application.
 
 {{- end }}
 
+{{- if not .Values.api.analytics.enabled }}
+
+######################################################
+#####      Analytics capabilities are off!       #####
+#####  Chart-provisioned InfluxDB is deprecated. #####
+######################################################
+
+If you were provisioning an InfluxDB instance for Flagsmith's
```

Comment:
I don't know that we need to worry about this but, just so I have the full picture... is there a way that users will be able to configure the charts such that their chart-provisioned influxdb will continue to be used? Also, I guess this throws the question about migration. We haven't written anything to migrate from InfluxDB to the Postgres backed analytics solution. The experience will actually be that they appear to lose all of the data they had in Influx once the API starts returning data from the postgres DB rather than Influx. So, you're right that the data isn't 'lost' but it will appear to be from the users' perspectives.

Reply:
> is there a way that users will be able to configure the charts such that their chart-provisioned influxdb will continue to be used?

I imagine there is, by configuring the chart-provisioned instance as an external one. Perhaps we should test and document this path.

> We haven't written anything to migrate from InfluxDB to the Postgres backed analytics solution.

True. No idea how hard would that be. I'm not sure there'd be high demand for this too, given that InfluxDB is still usable.